utilities/xmlrpcpp/test/CMakeLists.txt

```diff
@@ -34,11 +34,6 @@ if(TARGET test_dispatch_live)
   target_link_libraries(test_dispatch_live xmlrpcpp test_fixtures ${Boost_LIBRARIES})
 endif()
 
-catkin_add_gtest(test_ulimit test_ulimit.cpp)
-if(TARGET test_ulimit)
-  target_link_libraries(test_ulimit xmlrpcpp test_fixtures ${Boost_LIBRARIES})
-endif()
-
```

Comment:
Why did you remove this test?

Reply:
See the commit log of: https://github.com/ros/ros_comm/pull/2297/commits/600fedf1a4eb75752baab497d76c6b0642ec6068

This no longer works on Linux.